### PR TITLE
Support removing patterns and groups

### PR DESCRIPTION
**Pull request by jasedit**
This implements the very basic functionality for removing highlights and groups.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,6 +144,10 @@ Specifies a set of CSS properties to apply to a given group highlight. Multiple
 
 Reload highlight configuration.
 
+#### `:hilite remove <group|pattern> <text>`
```
Comment (ondreian) [inline README.md:147]: this should probably be `:hilite remove <group> <pattern|text>` since hilites can be regexp

Comment (jasedit) [reply README.md:147]: `<group|pattern>` is intended to convey that this is either the literal `group` or `pattern` to indicate which type of object is being removed. `<text` is meant to be the key in the relevant map, so either the regex pattern or the group name. Seems like this should be wordsmithed to be more clear however.

File: src/vimish/commands/hilite.js [hunk 2]
```diff
@@ -1,61 +1,104 @@
...
+          throw new Error(
+            `${pattern} is still in use. Please use :sudo :hilist remove group confirm <pattern> to force removal.`
+          )
+        }
+        pattern = pattern.slice(1)
+      }
+      Hilites.remove_group(pattern.join(" "))
+      break
+    case "pattern":
+      Hilites.remove_pattern(pattern.join(" "))
```
Comment (ondreian) [inline src/vimish/commands/hilite.js:76]: this should be refactored to use `return Hilites....` instead of dangling breaks

**Review by ondreian — APPROVED** [review]
lgtm, thanks

**Pull request merged**